.eslintrc.cjs

```diff
@@ -126,6 +127,12 @@ module.exports = {
 
         // Type conversions require an explicit cast in AssemblyScript.
         '@typescript-eslint/no-unnecessary-type-assertion': 'off',
+
+        // Disable jsdoc
+        'require-jsdoc': 'off',
+        'valid-jsdoc': 'off',
+        // Enable typedoc
+        "tsdoc/syntax": "error",
```

Comment:
single quote, double => make a choice and be consistent :laughing:

Reply:
good catch! :)